### ISSUE TITLE
wrong name or comment in economizer sequence

### ISSUE THREAD

**Issue by mwetter** [lbl-srg/modelica-buildings#854]
The blocks `EconModulationMultiZone` and `EconomizerMultiZone` have
```modelica
CDL.Interfaces.RealInput THeaSet(unit="K", quantity = "ThermodynamicTemperature")
    "Supply air temperature cooling setpoint"
```
e.g., one says heating, the other cooling.

Also, the figure in `EconomizerMultiZone` shows the wrong name for `ecoEnaDis`.

@milicag : Can you please address these points after https://github.com/lbl-srg/modelica-buildings/pull/832 passes all tests and is merged.

**Comment by milicag** [comment]
@mwetter I addressed this in PR #805. The merge should close this issue. It was an easy fix, since I've updated the variable name and forgot to update the comments, which I've now fixed.

**Comment by milicag** [comment]
This got resolved with #832.
